Sources/TuistGenerator/Linter/GraphLinter.swift

```diff
@@ -45,11 +45,33 @@ public class GraphLinter: GraphLinting {
         issues.append(contentsOf: lintMismatchingConfigurations(graphTraverser: graphTraverser))
         issues.append(contentsOf: lintWatchBundleIndentifiers(graphTraverser: graphTraverser))
         issues.append(contentsOf: lintCodeCoverageMode(graphTraverser: graphTraverser))
+        issues.append(contentsOf: lintWorkspaceUnuesedTargetsInWorkspace(graphTraverser: graphTraverser))
         return issues
     }
 
     // MARK: - Fileprivate
 
+    private func lintWorkspaceUnuesedTargetsInWorkspace(graphTraverser: GraphTraversing) -> [LintingIssue] {
+        graphTraverser.workspace.schemes.map { scheme in
+            let arr = scheme
+                .targetDependencies()
+                .filter { targetReference in
+                    if let target = graphTraverser
+                        .targets()[targetReference.projectPath],
```

Comment:
Can we create `let target = graphTraverser.targets()` at the top of the function so that we don't have to call `graphTraverser.targets()` for each item in the loop?

Reply:
Done it